examples/mysql_secure_test.js

```diff
@@ -0,0 +1,37 @@
+import sql from 'k6/x/sql';
+
+sql.loadTLS({
+  enableTLS: true,
```

Comment:
this seems redundant, is there a case when we want to loadTLS with `enableTLS: false`?

Reply:
As discussed, using this as a sentinel in `Open` to `RegisterTLSConfig`.